src/Admin/SearchAdmin.php

```diff
@@ -206,6 +206,11 @@ private function buildIndexedDocumentsList(): ArrayList
                     $query->where('ShowInSearch = 1');
                 }
 
+                // where appropriate, make sure we only show database total for this index
```

Comment:
On a project with indexing for subsites, it was noted that the total documents in the database didn't take into account the subsite (and index) to which the document related to.